walkthroughs/gpg_key_setup.md

```diff
@@ -0,0 +1,587 @@
+your prime keypair as its superior.
+
+To extend the landlord metaphor above, say you take your "prime" keys to your
+new apartment to the locksmith and get a set of duplicates made. You then take
+the prime keypair to the bank and put them in a safe deposit box. You then go
+out and, curse the luck!, you lose your duplicate keys.
+
+Instead of having to fear that someone found those keys, followed you home, and
+is waiting to come in and steal your comic books, you go to the bank, open up
+the safe deposit box, take out the prime keys and say "DESTROY DUPLICATE SET
+ONE!" and *poof* the lost keys vanish from existence! You then return to the
+locksmith, get a new pair made, return the prime pair, and then go on through
+life with your daily use duplicates.
+
+In public-key cryptography the stolen duplicates don't vanish, they are instead
+"revoked" and thus labeled "untrustworthy."
+
+### Sub-Step 8c: Secure the Prime Keypair
+
+* Put the _sub-key_ somewhere you can use it regularly
+* Put the "master" private key somewhere safe
+
+OK, enough prelude, let's make that "Prime" keypair.
+
+## Put GPG to Work
+
+### Step 9: Create the Public / Private Pair
+
+_As a supplementary reference, see [The GnuPG Manual](https://www.gnupg.org/gph/en/manual/c14.html)_
+
+In the terminal type:
+
+1. `cd`: make sure we start in your home directory
+1. `gpg --gen-key`: This will start the process of generating your "Prime" keypair
+2. Accept the default (1) RSA and RSA
+3. For keysize choose 4096. This will make your `master` key the most complex and hard to steal possible
+4. Set an expiration of "Never." The expiration date behaves the same as it
+does on credit cards and for the same purpose. Since this key will be protected as well as
+   possible and won't be taken into the world day-to-day, let's set this key to
+   be non-expiring. Similarly if you only left your credit card in a bank safe,
+   you could be rather comfortable with it having a distant expiry date. There
+   are arguments to why this shouldn't be done. A happy medium might be `5y` or
+   `10y` corresponding to five and ten years, respectively.
+5. Accept these changes
+6. Provide your real real name (here "Tutorial")
+7. Provide a valid email address (here `tutorial@example.com`)
+8. Leave comment blank
+9. Select **O**kay
+10. **SELECT A VERY STRONG PASSWORD** A weak password is the same as leaving your key under the mat. Make this very difficult to guess and / or store it in a password manager.
+11. Re-enter it to confirm
+
+Here's an example of what the output should look like:
+
+![Generated GPG Keypair](./keygen-walkthru-images/generated-prime-keypair.jpg)
+
+Here we see "Tutorial" has an identity of `7EC9E024`. If you see a much longer
+number beginning with `0x`, that's fine too!
+
+Henceforth in this guide will use these two identifiers ("Tutorial" and
+`7EC9E024`), but if you're typing along, use your name and key ID instead.
```

Comment:
What do you think about showing the command that will show them their key ID? That way they can check it or be reminded of it just before having to type it in any of the following commands.